### PR TITLE
Create Floyd_warshall.kt

### PR DESCRIPTION
**Pull request by kichloo**
Added Floyd Warshall algorithm in Kotlin language.

Fixes #2205  

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [ ] Sample Input / Output is added at the end of file.
- [ ] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added new file Floyd_warshall.kt

#### Languages Used:
- Kotlin
 

#### Files Added:
- Floyd_warshall.kt
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please include the issue number and let me know

**Comment by kichloo** [comment]
@asha15 The issue number is #2422

**Comment by kichloo** [comment]
@asha15 Done!

**Comment by asha15** [comment]
please give me link to your issue

**Comment by kichloo** [comment]
@asha15 The link is https://github.com/jainaman224/Algo_Ds_Notes/issues/2205
I am new to open source and I searched the issue from the list of issues. Is that how to do it?

**Comment by asha15** [comment]
just type the issue number next to `Fixes #` in  the PR template.

**Comment by kichloo** [comment]
@asha15 Done and thanks for the help!

**Comment by kichloo** [comment]
Done the changes. Pl check

File: Floyd_warshall.kt [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ 
+fun main(args: Array<String>) {
+    val weights = arrayOf(
+            intArrayOf(1, 3, -2),
+            intArrayOf(2, 1, 4),
+            intArrayOf(2, 3, 3),
+            intArrayOf(3, 4, 2),
+            intArrayOf(4, 2, -1)
+    )
+    val nVertices = 4
```
Comment (MastersAbh) [inline Floyd_warshall.kt:56]: I dont see dynamic inputs here

**Comment by MastersAbh** [comment]
Hi @kichloo are you working on this?
If not, please close the PR.

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this issue.

**Comment by kichloo** [comment]
I had done all the required changes but no one was there to review my PR. And in this case, dynamic input cannot be there.

**Comment by somya-kapoor** [comment]
Why? 2D array can be taken dynamically.

**Comment by kichloo** [comment]
I know but I got wrong output while taking the dynamic case, so fixed it as it is.

**Pull request closed without merging**